Block/BlockWateringCan.cs

```diff
@@ -88,6 +93,7 @@ public override void OnHeldInteractStart(ItemSlot slot, EntityAgent byEntity, Bl
             }
 
             handHandling = EnumHandHandling.PreventDefault;
+            base.OnHeldInteractStart(slot, byEntity, blockSel, entitySel, firstEvent, ref handHandling);

```

Comment:
this one looks out of place

Reply:
lockBehavior doesn't call OnHeldInteractStart because of this, which also makes custom interactions impossible

Reply:
you'd have multiple interactions running at the same time with this enabled, its not meant for this. It would create undefind ingame behavior. If you want to replace a blocks interact behavior you'll have to patch it